spec/controllers/admin/partners_controller_spec.rb

```diff
@@ -0,0 +1,48 @@
+RSpec.describe Admin::PartnersController, type: :controller do
+  let(:organization) { create(:organization) }
+
+  # Use the super_admin instead of organization_admin
+  let(:super_admin) { create(:super_admin) }
+
+  # Ensure partners are created before the test
+  let!(:partner2) { create(:partner, name: "Bravo", organization: organization) }
+  let!(:partner1) { create(:partner, name: "alpha", organization: organization) }
+  let!(:partner3) { create(:partner, name: "Zeus", organization: organization) }
+
+  let(:default_params) do
+    {organization_id: organization.id}
+  end
+
+  context "When logged in as a super admin" do
+    before do
+      sign_in(super_admin)  # Sign in as the super_admin
+    end
+
+    describe "GET #index" do
+      it "returns http success" do
+        get :index
+        expect(response).to be_successful
+      end
+
+      it "assigns partners ordered by name (case-insensitive)" do
+        get :index
+        expect(assigns(:partners)).to eq([partner1, partner2, partner3].sort_by { |p| p.name.downcase })
```

Comment:
You shouldn't need to sort since you know the partners' names. :) Just reorder the array to `partner2, partner1, partner3`.

Actually... can you switch it so the `partner1`line is moved above the `partner2` line? That way they'd actually be created out of order. The order of the `let!` statements is what matters.